src/main/java/dev/joopie/jambot/music/GuildMusicService.java

```diff
@@ -212,4 +221,16 @@ private void assertMemberInSameVoiceChannel(final GuildMusicPlayer musicPlayer,
             throw new JambotMusicServiceException("Join the voice channel!");
         }
     }
+
+    public void handleFeedback(final Guild guild, final String youtubeId) {
```

Comment:
Methode naam dekt niet de lading met wat het doet. "handle" is te algemeen. Het verwijdert een track obv youtube id uit de queue.